website/docs/docs/get-started/develop-in-the-cloud.md

```diff
@@ -47,6 +47,7 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
 | **File state indicators**  |  Ability to see when changes or actions have been made to the file. The indicators **M, D, A,** and **•** appear to the right of your file or folder name and indicate the actions performed: <br /> <br /> - Unsaved **(•)** &mdash; The IDE detects unsaved changes to your file/folder<br /> - Modification **(M)** &mdash; The IDE detects a modification of existing files/folders<br /> - Added **(A)** &mdash; The IDE detects added files<br/> - Deleted **(D)** &mdash; The IDE detects deleted files.
 | **IDE git button** | The git button in the IDE allows you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project. You can create or change branches, [resolve merge conflicts](/docs/collaborate/git/merge-conflicts), and execute git commands directly in the IDE. |
 | **Documentation** | You can generate and view your [project documentation](/docs/collaborate/build-and-view-your-docs) for your dbt project in real-time. You can inspect and verify what your project's documentation will look like before you deploy your changes to production. |
+| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. <br /> <br /> The **Preview** button will display 500 rows by default, but you can change this by adding `limit your_number` at the end of your SQL statement. For example, `SELECT * FROM table limit 100` will return up to 100 rows.  Remember that you must write the `limit your_number` explicitly and cannot derive it from a macro. |
```

Comment:
I know this is already out in the wild - sorry for just now looking at it! 

One note is that `Compile` does not actually hit the data platform. Compile happens totally local within dbt.

Reply:
ah thanks so much for clarifying that @davidharting and I'll get a pr up to clarify this 🙏

Reply:
hey @davidharting , i opened up [this pr](https://github.com/dbt-labs/docs.getdbt.com/pull/3021) to clarify the above. would you be able to take a look to see if i captured it correctly? thank you!

Reply:
thank you! Just gave that PR a ✅